test/scenarios/bookshop/funcs.test.js

```diff
@@ -234,7 +260,8 @@ describe('Bookshop - Functions', () => {
     })
 
     test.skip('fractionalseconds', async () => {
-      // REVISIT: ERROR: Feature is not supported: Method "fractionalseconds" in $filter or $orderby query options
+      // okra error: Feature is not supported: Method "fractionalseconds" in $filter or $orderby query options
+      // new adapter: no error, wrong result
```

Comment:
The function does not seem to produce the wrong result, but it is a miss match of precision. The function returns with 7 precision, but the query is checking for 3 precision.

[fractionalseconds](https://github.com/cap-js/cds-dbs/blob/main/db-service/lib/cql-functions.js#L197C44-L197C50)